groupthink/commands.py

```diff
@@ -13,26 +15,37 @@
     dest = '/usr/local/bin'
 
 
+@arg('org', help='The org to install')
+@arg('-a','--alias', help='Set a different name for this command')
+@arg('-d','--dest')
+@arg('-s', '--storage')
 def install(org, alias=None, dest=dest, storage=storage):
-    # install organization scripts repository in ~/.groupthink
+    """
+    Provided a GitHub org name, installs that org's CLI scripts.
+    """
+    # install org scripts repository in ~/.groupthink
     # and add groupthink script to /usr/local/bin
     alias = alias if alias else org
-    check_install(alias, check_already=False, storage=storage)
+    check_install(org, check_already=False, storage=storage)
     script_exists = os.path.exists('{dest}/{alias}'.format(dest=dest, alias=alias))
     if script_exists:
         message = []
         message.append('A script is already installed with the name {alias}.'.format(alias=alias))
         message.append('You can try to install {org}-cli under a different name, with:\n'.format(org=org))
-        message.append('  groupthink --install {org} --alias name'.format(org=org))
+        message.append('  groupthink --install {org} --alias [alias]'.format(org=org))
         return '\n'.join(message)
     try:
         execute_cmd(['mkdir', '-p', storage])
         execute_cmd(['mkdir', '-p', dest])
         repo_url = 'https://github.com/{org}/{org}-cli.git'.format(org=org)
         repo_dest = '{storage}/{alias}-cli'.format(storage=storage, alias=alias)
-        (msg, err) = execute_cmd(['git', 'clone', repo_url, repo_dest])
-        if err.find('fatal: ') > -1:
-            sys.exit(1)
+        if requests.get(repo_url[:-4]).status_code < 400:
```

Comment:
The reason for the check is that, when the repo doesn't exist, `git` prompts the user for GitHub credentials. I think this is because a private repo would return a 404 if unauthenticated but could still be cloned with proper credentials.